gitprompt.sh

```diff
@@ -346,7 +346,13 @@ function setGitPrompt() {
   unset GIT_PROMPT_SHOW_UNTRACKED_FILES
 
   if [[ -e "$repo/.bash-git-rc" ]]; then
-    source "$repo/.bash-git-rc"
+    # The config file can only contain variable declarations on the form A_B=0 or G_P=all
+    local CONFIG_SYNTAX="^(FETCH_REMOTE_STATUS|GIT_PROMPT_SHOW_UNTRACKED_FILES|GIT_PROMPT_IGNORE)=[0-9a-z]+$"
```

Comment:
Ok, according to https://github.com/magicmonty/bash-git-prompt/issues/324#issuecomment-298931802 your solution seems the safer one